libfwupd/fwupd-common.c

```diff
@@ -809,28 +809,39 @@ fwupd_guid_from_string(const gchar *guidstr,
 gchar *
 fwupd_guid_hash_data(const guint8 *data, gsize datasz, FwupdGuidFlags flags)
 {
-	const gchar *namespace_id = FWUPD_GUID_NAMESPACE_DEFAULT;
 	gsize digestlen = 20;
 	guint8 hash[20];
-	fwupd_guid_t uu_namespace;
 	fwupd_guid_t uu_new;
 	g_autoptr(GChecksum) csum = NULL;
+	const fwupd_guid_t uu_default = {0x6b,
+					 0xa7,
+					 0xb8,
+					 0x10,
+					 0x9d,
+					 0xad,
+					 0x11,
+					 0xd1,
+					 0x80,
+					 0xb4,
+					 0x00,
+					 0xc0,
+					 0x4f,
+					 0xd4,
+					 0x30,
+					 0xc8};
+	const fwupd_guid_t uu_microso = {0x70, 0xff, 0xd8, 0x12, 0x4c, 0x7f, 0x4c, 0x7d};
```

Comment:
uu_microsoft perhaps?

Reply:
I wanted it to align with the other one. Is that insane?

Reply:
no worry.